taskcluster/.shared.yml

```diff
@@ -4,7 +4,7 @@ python:
   packages_stretch:
     apt: 'make build-essential libssl1.0-dev zlib1g-dev libbz2-dev libreadline-dev libsqlite3-dev curl llvm libncurses5-dev libncursesw5-dev xz-utils tk-dev'
   brew:
-    setup: 'install_local_homebrew "python-ds-test" && install_pkg_local_homebrew "sox"'
+    setup: 'install_local_homebrew "python-ds-test" && install_pkg_local_homebrew "sox" && install_pkg_local_homebrew "readline" && install_pkg_local_homebrew "openssl" && install_pkg_local_homebrew "pkg-config"'
```

Comment:
Looks like we might have some spurious case of build not picking the proper OpenSSL/readline versions, between the one from system, etc., so install it in brew for sure. Python's build would rebuild-it anyway, so we might win from some brew caching here.